### ISSUE TITLE
Controllers don't work

### ISSUE THREAD

**Issue by MelGipson** [alvr-org/ALVR#2586]
## Description

ALVR does not recognize controllers, but when I'm put them  down, handstracking is works. I tried different controller settings but couldn't get it to work
I'm use Pico 4 Ultra

## General Troubleshooting
- [x] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [x] I read the [ALVR GitHub Wiki](https://github.com/alvr-org/ALVR/wiki)

### Hardware
**Note**: for Linux, an upload to the [`hw-probe`](https://linux-hardware.org/) database is preferred: `hw-probe -all -upload`

**CPU**: Intel(R) Core(TM) i5-12600K

**GPU**: NVIDIA GeForce RTX 4060

**GPU Driver Version**: 566.14

**Audio**:  NVidia High Definition Audio

### Installation
**ALVR Version**: 20.11.1, nightly

**ALVR Settings File**:

**SteamVR Version**: 2.8.8

**Install Type**:
- [ ] Packaged (`exe`, `deb`, `rpm`, etc)
-   Portable (`zip`)
- [ ] Source

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions): Windows 10 Pro 22H2

**Comment by The-personified-devil** [comment]
Does this issue occur only when streaming or also in the alvr lobby?

**Comment by MelGipson** [comment]
> Does this issue occur only when streaming or also in the alvr lobby?
In the alvr lobby too

**Comment by MrHills-2** [comment]
Same problem. I play DCS and the moment i activate the controllers the game crashes. Hand tracking works fine tho.

**Comment by zmerp** [comment]
@MrHills-2 I have a Pico 4 ultra but unfortunately I cannot reproduce. Are you experiencing this after a recent firmware update?

**Comment by MrHills-2** [comment]
Not really, i just started playing with ALVR, I have no prior experience. I decided to open an issue about it tho with more info. Should I try older versions of alvr to see if it used to work? Specifically there's a particular version i should try?

https://github.com/alvr-org/ALVR/issues/2590

**Comment by 0Human0** [comment]
Nightly should be working. Stable, however, is before the Pico 4 ultra controller fix. Are you sure that you tried/installed Nightly correctly?

**Comment by MrHills-2** [comment]
I'm using ALVR Launcher, i tried both of these
![2024-12-23-171245_screenshot](https://github.com/user-attachments/assets/469c9a93-3ecd-4a96-a09e-e7ecbd9d5b29)

**Comment by 0Human0** [comment]
Oh normal Pico 4, You did say ultra at the beginning but alright. What firmware version are you on?

**Comment by MrHills-2** [comment]
5.11.2.S. It says my system is up to date. Yeah sorry I didn't notice OP had an Ultra. Tbh i wasn't aware of an ultra version in the first place.

**Comment by 0Human0** [comment]
My bad I wasn't paying attention to names

**Comment by 0Human0** [comment]
I'm on that same firmware version and stable and up works just fine on my Pico 4

**Comment by MrHills-2** [comment]
Maybe it's a DCS on linux issue then. It's not officially supported after all.

**Comment by zmerp** [comment]
@MrHills-2 In ALVR nightly, in the passthrough view with the white text, you should see white crossairs to mark the position of the controllers. if you instead only see the wireframe skeleton of your hands, press some controller buttons to wake up the controllers.

**Comment by byhal** [comment]
Recently started experiencing this on my Quest 2, in SteamVR and lobby latest nightly

**Comment by The-personified-devil** [comment]
@byhal Can you provide any further information? Does this occur in the lobby and when streaming? Does it occur with the latest nightly? etc.

**Comment by byhal** [comment]
@The-personified-devil All of them, yes. When I go into a different app in my headset it'll do that

**Comment by curoviyxru** [comment]
Could the lack of a `/interaction_profiles/bytedance/pico4s_controller` controller profile be the reason of the original issue with Pico 4 Ultra? I guess Pico has allowed runtime to work with controllers via Pico 4 profile in 5.12.0U.
